src/utils/index.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Reverses the order of bytes in a buffer.
+ * @param buffer - The buffer to reverse.
+ * @returns A new buffer with the bytes reversed.
```

Comment:
It looks like this function modifies the input buffer as well. We should note that as a warning and the docstring should not specify that the buffer returned is a "new" one.